backend/controllers/selectResidentialLocation.ts

```diff
@@ -1,15 +1,21 @@
 import moment from 'moment'
 import { getCurrentPeriod } from '../utils'
 
-export default (whereaboutsApi) => {
+export default (locationsInsidePrisonApi, systemOauthClient) => {
   const renderTemplate = async (req, res, pageData?) => {
     const today = moment()
     const { errors, formValues } = pageData || {}
     try {
       const {
         user: { activeCaseLoad },
       } = res.locals
-      const residentialLocations = await whereaboutsApi.searchGroups(res.locals, activeCaseLoad.caseLoadId)
+
+      const systemContext = await systemOauthClient.getClientCredentialsTokens(req.session.userDetails.username)
+
```

Comment:
The key change in this PR

Reply:
how the secrests are managed in this project?